stages/org.osbuild.pacman.conf

```diff
@@ -15,24 +15,68 @@ SCHEMA = """
     "type": "string",
     "description": "pacman architecture",
     "default": "x86_64"
+  },
+  "mirrors": {
+    "type": "array",
+    "items": {
+      "type": "string"
+    }
+  },
+  "repositories": {
+    "type": "array",
+    "items": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["name"],
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "siglevel": {
+          "type": "string"
```

Comment:
Could be an enum to both restrict and document the valid options.
```json
"enum": ["Required", "Optional", "Never", "TrustAll", "TrustedOnly"]
```
Is that correct?

Reply:
That sounds good, but `SigLevel` has multiple valid values hmm, so maybe it should be an array?:
```
           SigLevel = Optional TrustedOnly
```

```json
"siglevel": {
   "type": "array",
   "items": {
   },
   maxItems: 2,
}
```

Sounds much better!

Reply:
ah interesting.  Then it would be an array of enums?  Sure sounds good :)